### ISSUE TITLE
Comments on software

### ISSUE THREAD

**Issue by fmu2** [gitter-lab/ml4bio#35]
- [x] Can we give a visual indication that there are three stages to advance through as the user selects data, trains models, and tests a model?  Perhaps show these as tabs?
- [x] How does the user know that they should wait for training to finish?  For a large dataset, will it look like the GUI hung?
- [ ] I received this warning, which should be fixed in the next scikit-learn release so we can ignore it https://stackoverflow.com/questions/48687375/deprecation-error-in-sklearn-about-empty-array-without-any-empty-array-in-my-cod
- [ ] We should start to look for functions and parts of the code that would support unit tests.
- [ ] Can I save the best classifier, either the trained model or the description of its parameters?  Can I save the table of models I tried and their performance?  This may be out of scope for the initial software but a potential v2 feature.
- [x] Can we have larger font sizes in the plots?
- [x] Is there a way for us to define jargon like classifier hyperparameters within the GUI?  Or should we link to sklearn docs?
- [x] After we have the workshop guides ready, please document the code organization so that others could eventually help maintain it.
- [x] Should we display the class label (column name or valid values) in the Data Summary panel?
- [x] After the warning about too many samples for leave-one-out, the software still advanced to the next tab.  Is that intended behavior?  What is that sample threshold?
- [x] Is the Data Plot implemented or is it a placeholder?
- [x] In the iris dataset I had a classifier that reported training AUROC of 1 in the table and figure legend but the ROC plot was not a flat line.  See the attached image.
- [ ] Need better documentation or more robust file loading for the unlabeled data csv file.  Should we accept a file that has class labels and pop up a warning that the labels have been removed?
- [ ] When saving the results `.csv` should be added to the end of the file name automatically if it isn't there already. 
- [ ] The default iterations for the neural network is too low.  On toy dataset 1, it reaches the max iterations and fails training on default settings. 
- [ ] Implement plot figure saving.
- [x] Document expected warnings from the wrapper scripts.
